static/bonecard.js

```diff
@@ -116,8 +116,9 @@ function init() {
 
             var i = 0;
             $.each(data.files, function(index, val) {
-                title = val.filename.substring(13);
-                card_type = val.filename.substring(8, 12);
+                bonecard_index = val.filename.indexOf("bonecard");
```

Comment:
This feels like a bit of an arbitrary compatibility change and provides a very limited and not well-specified format. If you need multiple data fields, I suggest simply leaving this to be a title and create a .JSON file with the additional fields.